s3/s3.py

```diff
@@ -0,0 +1,13 @@
+from s3fs import S3FileSystem
+
+
+def connect(key: str, secret: str) -> S3FileSystem:
+    """Create a Boto3 client for S3 service."""
+    filesystem = S3FileSystem(key=key, secret=secret)
+    return filesystem
+
+def save_data(filesystem: str, bucket: str, data: str) -> dict:
+    """saves the data received from AI Service"""
```

Comment:
Capital at the beginning of the docstring and dot at the end :wink:  Nit picks, I know...